bucketlist/app.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask
```

Comment:
Arrange your imports alphabetically and separate python file imports from own-generated files imports